### DISCUSSION TITLE
Raycast doesn't have permission to open Kitty?

### DISCUSSION THREAD

**Discussion by workadev** [kovidgoyal/kitty#8056] [Q&A]
Hi, I can't seem to launch Kitty from Raycast.
It says Raycast does not have permission to open `"(null)"`
<img width="332" alt="Screenshot 2024-11-18 at 19 17 24" src="https://github.com/user-attachments/assets/9a2a5e19-4ac1-45ee-a394-87540607113e">

I'm on nix darwin and enabled Kitty from home-manager. Might got something to do with it. Searched for similar issue but couldn't find an answer.

Launching from terminal is fine (kitty 0.37.0).

**Comment by kovidgoyal** [comment]
Use the official kitty binaries instead of the nix package and you will be fine.
